mathics/builtin/graphics.py

```diff
@@ -2602,7 +2602,7 @@ def to_svg(self):
         # svg = (
         #    '<foreignObject x="%f" y="%f" ox="%f" oy="%f" style="%s">'
         #    "<math>%s</math></foreignObject>")
-        svg = ('<text x="%f" y="%f" ox="%f" oy="%f" style="%s">' "%s" "</text>") % (
+        svg = ('<text x="%f" y="%f" ox="%f" oy="%f" style="text-anchor:middle; dominant-baseline:middle; %s">' "%s" "</text>") % (
```

Comment:
I suspect to do a good job a function is going to have to get called width and hight information and maybe other information about the SVG.

Reply:
This is the main change I made here: just use boxes_to_text instead boxes_to_xml.

Reply:
> 
> 
> I suspect to do a good job a function is going to have to get called width and height information and maybe other graphics information that might be in the SVG in complicated form but in the Mathics Graphics object in less complicated form.

I may be wrong here. Possibly https://github.com/mathics/Mathics/blob/5aa0367be03d9e912f5c082fae33abeec87b1650/mathics/builtin/graphics.py#L3355-L3356 is the culprit. The size should be determined by the number of items in xmin to xmax and ymin to ymax.

Reply:
@mmatera I spent some time looking again at this, and I am now pretty sure that it is not every "InsetBox" that you want to change but instead the thing that is calling this which is called "create_axes".

Reply:
Actually, using text or mathml should depend on what is the Expression: for graphics objects (points, lines, disks,...) should be mathml or svg while for anything else, text.

Reply:
@rocky, my last attempt on this:  ``Inset`` was not implemented, except in the case of strings and expressions (labels, including axes labels). With the current implementation, MathML support is not possible. I added reduced support for ``Graphics`` insets, but full support would require more work than what I plan to invest in this. 
If this works, the next step is to work out the MPL package, in a way that all that we have here will be implemented there.